mix.exs

```diff
@@ -58,4 +41,32 @@ defmodule OMG.Umbrella.MixProject do
       {:ex_doc, "~> 0.19", only: :dev, runtime: false}
     ]
   end
+
+  defp aliases do
+    [
+      test: ["test --no-start"],
+      coveralls: ["coveralls --no-start"],
+      "coveralls.html": ["coveralls.html --no-start"],
+      "coveralls.detail": ["coveralls.detail --no-start"],
+      "coveralls.post": ["coveralls.post --no-start"],
+      "ecto.setup": ["ecto.create", "ecto.migrate", "run apps/omg_watcher/priv/repo/seeds.exs"],
+      "ecto.reset": ["ecto.drop", "ecto.setup"]
```

Comment:
I think `ecto` tasks won't work run from repo root

Reply:
I tested the `watcher` + `child_chain`setup and it seemed to work but I will check again to be 100% sure.

Reply:
It's working like this :)